docs/Reference/CLI/CLI-Subcommands.md

```diff
@@ -174,4 +174,26 @@ This data is included in the [IBFT 2.0 genesis file](../../HowTo/Configure/Conse
         
         ``` tab="RLP Output"
         0xf853a00000000000000000000000000000000000000000000000000000000000000000ea94be068f726a13c8d46c44be6ce9d275600e1735a4945ff6f4b66a46a2b2310a6f3a93aaddc0d9a1c193808400000000c0
-        ```
\ No newline at end of file
+        ```
+
+## retesteth
+
+This command enables Retesteth-compatibilty, and accepts the following command line options:
+
+* [\--data-path](../CLI-Syntax#data-path)
+* [\--host-whitelist](../CLI-Syntax#host-whitelist)
+* [\--rpc-http-host](../CLI-Syntax#rpc-http-host)
+* [\--rpc-http-port](../CLI-Syntax#rpc-http-port)
+* [\--logging](../CLI-Syntax#logging)
+
+!!! note
```

Comment:
I'd move this into the first paragraph (ie, after _Runs a Retesteth compatible server_) - mainly because I think it's information that's relevant to understanding why you'd use it rather than a side note.